includes/Abilities/StockImageAbilities.php

```
@@ -196,16 +195,61 @@ private static function download_and_import( string $keyword, int $width, int $h
 			require_once ABSPATH . 'wp-admin/includes/image.php';
 		}
 
-		$tmp_file = download_url( $url, 30 );
+		$last_error = '';
+		$tmp_file   = null;
+		$image_url  = '';
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$image_url'. (undefined)

(UnusedLocalVariable)